Improbable/Stdlib/Improbable.Stdlib/WorkerConnection.cs

```diff
@@ -600,7 +620,7 @@ private static async Task<double> GetCpuUsageForProcess(CancellationToken cancel
         {
             var startTime = DateTime.UtcNow;
             var startCpuUsage = Process.GetCurrentProcess().TotalProcessorTime;
-            await Task.Delay(500, cancellationToken);
+            await Task.Delay(500, cancellationToken).ConfigureAwait(false);
```

Comment:
```suggestion
            await Task.Delay(TimeSpan.FromSeconds(0.5), cancellationToken).ConfigureAwait(false);
```

Or the `FromMillis` one :)